evento/src/views/Explore/index.js

```diff
@@ -4,21 +4,37 @@ import EventCard from '../../components/EventCard';
 import './Explore.css';
 
 class Explore extends Component {
-	componentWillMount() {
-		this.setState({ events: [] });
+	constructor(props) {
+		super(props);
+		this.state = { events: [] };
+	}
 
+	componentDidMount() {
 		fetch('/events')
 		.then(response => response.json())
 		.then(events => {
 			this.setState({ events: events });
 		});
 	}
+	
+	isVisible(event, filterQuery) {
```

Comment:
Also I really don't think that this logic belongs here. The same logic would have to be copied to my events page for example and this is restrictive. What if we would like to filter to only show category "Sports", or maybe to show only the events in the next few days or something? I think that  much better way would be that Explore gets a function (either this.props.filter.match(event) or this.props.filter(event)) so that Explore itself doesn't have any knowledge on how the filtering is implemented.